src/ykfde-open

```diff
@@ -137,10 +160,18 @@ if [ "$YKFDE_PRINT_ONLY" ]; then
   exit 0
 fi
 
+if [ "$TEST_PASSPHRASE" ]; then
+  [ "$DBG" ] && printf '%s\n' " > Passing '$YKFDE_PASSPHRASE' to 'cryptsetup'"
+  [ "$DBG" ] && printf '%s\n' " > Decrypting with 'cryptsetup luksOpen $TEST_PASSPHRASE $LUKS_KEYSLOT $YKFDE_LUKS_DEV'..." || echo " > Decrypting with 'cryptsetup'..."
+  printf %s "$YKFDE_PASSPHRASE" | cryptsetup luksOpen "$TEST_PASSPHRASE" "$LUKS_KEYSLOT" "$YKFDE_LUKS_DEV" 2>&1
+  printf '%s\n' "   Device successfully opened"
+  exit 0
+fi
+
 if [ "$(id -u)" -eq 0 ]; then
   [ "$DBG" ] && printf '%s\n' " > Passing '$YKFDE_PASSPHRASE' to 'cryptsetup'"
-  [ "$DBG" ] && printf '%s\n' " > Decrypting with 'cryptsetup luksOpen $YKFDE_LUKS_DEV $YKFDE_LUKS_NAME $YKFDE_LUKS_OPTIONS'..." || echo " > Decrypting with 'cryptsetup'..."
-  printf %s "$YKFDE_PASSPHRASE" | cryptsetup luksOpen "$YKFDE_LUKS_DEV" "$YKFDE_LUKS_NAME" "$YKFDE_LUKS_OPTIONS" 2>&1
+  [ "$DBG" ] && printf '%s\n' " > Decrypting with 'cryptsetup luksOpen $LUKS_KEYSLOT $YKFDE_LUKS_DEV $YKFDE_LUKS_NAME $YKFDE_LUKS_OPTIONS'..." || echo " > Decrypting with 'cryptsetup'..."
+  printf %s "$YKFDE_PASSPHRASE" | cryptsetup luksOpen "$LUKS_KEYSLOT" "$YKFDE_LUKS_DEV" "$YKFDE_LUKS_NAME" "$YKFDE_LUKS_OPTIONS" 2>&1
```

Comment:
When `$LUKS_KEYSLOT` is not set this fails. Please move it as last param: `"$YKFDE_LUKS_DEV" "$YKFDE_LUKS_NAME" "$YKFDE_LUKS_OPTIONS" "$LUKS_KEYSLOT"`. Do the some to debug line above for consistency.